shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/schema/RuleSchemaMetaDataTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.metadata.schema;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class RuleSchemaMetaDataTest {
+    @Test
+    public void assertNewInstance() {
+        RuleSchemaMetaData obj = new RuleSchemaMetaData(null, null);
+        assertNotNull("RuleSchemaMetaData is null", obj);
+    }
```

Comment:
The assertion of new instance does not make sense.
Please add test case for `getSchemaMetaData`

Reply:
ok